src/components/NavBar/NavBarSkipLink.jsx

```diff
@@ -0,0 +1,63 @@
+// React Imports
+import React, { useState } from 'react';
+
+// MUI theme
+import theme from '../../theme';
+
+/**
+ * NavBarSkipLink component is an accessibility component
+ * that allows users to skip to the main content of the page.
+ *
+ * @returns {React.Element} The rendered NavBarSkipLink component.
+ */
+const NavBarSkipLink = () => {
+  // state used for hiding and showing component
+  const [animateOut, setAnimateOut] = useState(true);
+
+  // when TAB key is pressed and bring link into focus
+  const handleFocus = () => {
+    setAnimateOut(false);
+  };
+
+  // handles blur on link
+  const handleBlur = () => {
+    setAnimateOut(true);
+  };
+
+  // handles clicking on link and scrolling to content
+  const handleClick = (event) => {
+    event.preventDefault();
+    setAnimateOut(true);
+    const targetElement = document.getElementById('main-content');
+    if (targetElement) {
+      targetElement.scrollIntoView({ behavior: 'smooth' });
+    }
```

Comment:
Depending on the resulting behavior of a React Router solution, you probably won't need to have this either.

I'd also say that if we should delegate scroll behavior to the user or the site itself. If we have smooth scroll behavior set for the whole site (or not), we should use that setting.